tests/testthat/helper.R

```diff
@@ -2,14 +2,16 @@ transition_example_a <- function() {
   f <- function(x, y, a) a
   environment(f) <- emptyenv()
 
-  new_transition(
+  transition(
     from = "a",
     to = "b",
     transition_type = "probability",
     mortality_type = NULL,
     fun = new_transition_function(f),
-    predictors = c(x = "temp", y = "host_density"),
-    parameters = new_parameters(a = 1)
+    predictors = list(
```

Comment:
here's an example of the new `predictor_spec` in use

Reply:
Great!